samza-kafka/src/main/scala/org/apache/samza/system/kafka/KafkaSystemConsumer.scala

```diff
@@ -217,11 +217,11 @@ private[kafka] class KafkaSystemConsumer(
               case _ => debug("Ignoring refresh for %s because we already added it from another thread." format head)
             }
           }
-          rest
+          tpToRefresh.drop(1)
```

Comment:
Minor: Slightly cleaner to do tpToRefresh.tail. Is that equivalent here?

Reply:
sure, that works too. Let me change to it.